lib/queue_classic_plus/inflector.rb

```diff
@@ -0,0 +1,13 @@
+module QueueClassicPlus
+  module Inflector
+    def self.underscore(camel_cased_word)
```

Comment:
Was this mostly taken from somewhere else? If yes, please add the source as a comment, could be useful in the future.